### DISCUSSION TITLE
Disable cron for an app

### DISCUSSION THREAD

**Discussion by turicas** [dokku/dokku#6947] [Ideas]
There are some use cases where I do not want the app's cronjobs to run, like in data migrations (I have cronjobs that change the database and this would disrupt the migration). I couldn't find a simple way to do it, since `dokku ps:stop appname` and `dokku maintenance:enable appname` do not affect cron (I've checked by running `tail -500f /var/log/syslog | grep 'dokku run --cron-id'` after stopping/putting the app into maintenance).
Since I don't want to make new deploys changing `app.json`, I'm currently commenting the cronjobs manually using `sudo -u dokku crontab -e` and uncommenting after the migration.
Is there any other clever way to do it or is it a missing feature?

It's somewhat related, but not exactly the same as https://github.com/dokku/dokku/discussions/5024

**Comment by josegonzalez** [comment]
This is probably best set as a feature of dokku vs anything else. Mind opening an issue?

**Reply by turicas** [reply]
Thanks! I've created the issue: #6953
